guides/source/getting_started.md

```diff
@@ -1129,8 +1129,10 @@ you attempt to do just that on the new article form
 We've covered the "CR" part of CRUD. Now let's focus on the "U" part, updating
 articles.
 
-The first step we'll take is adding an `edit` action to the `ArticlesController`,
-generally between the `new` and `create` actions, as shown:
+The first step we'll take is adding an `edit` action to the 
```

Comment:
Nice! Maybe move the "Keep in mind that.." part to a seperate sentence after "generally 
between the `new` and `create` actions, as shown" ?

Or maybe even move it to a yellow note block?
Like https://guides.rubyonrails.org/active_record_basics.html#object-relational-mapping
```
The first step we'll take is adding an `edit` action to the  `ArticlesController`,
generally between the `new` and `create` actions, as shown.

NOTE:  Keep in mind that we're using `edit` to render a view. For the actual updating  we'll add an
`update` action later.
```

Reply:
Yes, lets move to a NOTE. Its getting more harder to read this.
I was on the fence for this addition, but we can add and see how this improves the reading.

Reply:
Maybe also change: 
```
For the actual updating function, we'll add an `update` action later.
```
To
```
For the actual saving of the changes to the Article, we'll add an `update` action later.
```
To make it more clear it's saving changes to the database.

Reply:
This is a pretty good suggestion! 

I didn't think about adding a yellow "note" block, because I didn't think this part should raise big attention for readers (and I didn't know how to write a yellow note block by code either :sweat_smile: ). 

Frankly speaking, I think appending the `edit` method to the code example block shall be good enough to let readers notice the difference between `update` and `edit`, but it won't harm to add a little explanation in the text. 

And this
```
For the actual saving of the changes to the Article, we'll add an `update` action later.
```
Makes things well explained!

Reply:
I have updated those ideas with a new commit, how does it feel like now?